document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresClient.java

```diff
@@ -51,6 +51,23 @@ public Connection getPooledConnection() throws SQLException {
     return connectionPool.getConnection();
   }
 
+  public void close() {
+    if (connection != null) {
+      try {
+        log.info("Closing connection #{} to {}", count, connectionConfig.toConnectionString());
```

Comment:
Do we need this info message?

Reply:
The existing method moved here since it was made public. Maybe, worth considering making it as `debug`?